mongo_connector/connector.py

```diff
@@ -1062,21 +1062,33 @@ def import_dm_by_path(path):
 
     def apply_ssl(option, cli_values):
         option.value = option.value or {}
-        ssl_certfile = cli_values.pop('ssl_certfile')
-        ssl_keyfile = cli_values.pop('ssl_keyfile')
-        ssl_cert_reqs = cli_values.pop('ssl_cert_reqs')
-        ssl_ca_certs = (
-            cli_values.pop('ssl_ca_certs') or option.value.get('sslCACerts'))
+        ssl_certfile = cli_values.get('ssl_certfile')
```

Comment:
dict.get takes a "default" value, so you can do:

ssl_certfile = cli_values.get('ssl_certfile', cli_values.get('sslCertfile'))